src/meta/src/barrier/mod.rs

```diff
@@ -301,6 +301,7 @@ where
                     }
                 }
             }
+            min_interval.tick().await;
```

Comment:
cc @BugenZhao is this change okay?

Reply:
I've done several runs on my server, and it looks like everything still work without this await.

Reply:
Why we need this delay?

Reply:
I will get the same epoch in my macbook if do not delay the epoch generation.

Reply:
Well, let's remove this later after monolithic clock has been implemented?

Reply:
The epoch generate interval is by default 100ms, seems like this should not gonna happen? 🤔

Reply:
This might be a problem with the epoch generator? I think it should not be hacked on the barrier manager.

Reply:
But there must be some problem because the epoch shall not be generated so quickly.

Reply:
+1 for keep this after monotonic clock has been implemented. Plz add a TODO or FIXME here.